android/BOINC/app/src/main/java/edu/berkeley/boinc/rpcExtern/authenticate/Hex.kt

```diff
@@ -0,0 +1,26 @@
+package edu.berkeley.boinc.rpcExtern.authenticate
+
+class Hex {
+    val hexArray = "0123456789abcdef".toCharArray()
+    fun HexToByte(hexChars: String) : ByteArray {
```

Comment:
You could make these extension functions instead.

Reply:
Maybe, but I liked to separate the new code. This is tested any change is not